tests/activities_sticky_queues/activities_sticky_queues_activity_test.py

```diff
@@ -0,0 +1,41 @@
+import uuid
```

Comment:
You don't have to test these FS functions if you want, what you had before is just fine. The only tests I was thinking was to test the workflow behavior (can even mock out the activities w/ different definitions here).

Reply:
I might hang on to them seeing as they're already written, I was using them to help me check some refactoring decisions anyway